libgnucash/engine/kvp-frame.hpp

```diff
@@ -226,10 +226,11 @@ struct KvpFrameImpl
     bool empty() const noexcept { return m_valuemap.empty(); }
     friend int compare(const KvpFrameImpl&, const KvpFrameImpl&) noexcept;
 
+    KvpFrame * get_child_frame_or_nullptr (Path const &) noexcept;
```

Comment:
Why, since you don't use it?

Reply:
It would be used in https://github.com/Gnucash/gnucash/pull/667/commits/2f6c89377e9c690ba08ed863fe24ca82c9591c1f#diff-a2fe08453182373caa62b6e813e7050bd5944a14592a1e1f0b9b9607be8e7605R4737 -- but this branch may be mothballed -- not sure how useful it'll be _if_ a reconciliation data structure is introduced in the future.

Reply:
Mothballed or not, don't use KVP directly, use `qof_instance_get` instead.